dist/tools/flake8/check.sh

```diff
@@ -43,7 +45,19 @@ ${FLAKE8_CMD} --version &> /dev/null || {
     exit 1
 }
 
-ERRORS=$(${FLAKE8_CMD} --config="${RIOTTOOLS}"/flake8/flake8.cfg ${FILES})
+ERRORS=$(${FLAKE8_CMD} --config="${RIOTTOOLS}"/flake8/flake8.cfg ${FILES} | ${LOG})
+
+if github_annotate_is_on; then
+    grep "^.\+:[0-9]\+:" "${LOGFILE}" | while read line; do
```

Comment:
You can parse `${ERRORS}` here directly without taking the detour the log file:
```suggestion
    grep "^.\+:[0-9]\+:" "${ERRORS}" | while read line; do
```

Reply:
I tried this initially but maybe I didn't wait long enough for the annotations to show up, lets seeeeeeeee

Reply:
Yeah, they only show up, after the run is finished.

Reply:
Mhhh something still doesn't work. Let me check locally.